openapi/controller/pinguin.py

```diff
@@ -0,0 +1,461 @@
+# -*- coding: utf-8 -*-
+"""Pinguin module for Odoo REST Api.
+
+This module implements plumbing code to the REST interface interface concerning
+authentication, validation, ORM access and error codes.
+
+It also implements a ORP API worker in the future (maybe).
+
+Todo:
+    * Implement API worker
```

Comment:
I'd prefer to move essential todos to README

Reply:
It's because of: http://www.sphinx-doc.org/en/master/ext/example_google.html